places-solution-integrations/analytics/attach-data.md

```diff
@@ -0,0 +1,70 @@
+# Attaching Places Data to Analytics Events via Launch Rules
+
+This document assumes that you have Adobe Places implemented in your application. If you need help implementing Adobe Places, see [Places Extensions for AEP SDK](../../configure-places-in-the-sdk/README.md).
+
+Once Places is sending in events for entries and exits, you can leverage Rules in Launch to attach your Places data to all Adobe Analytics events. With your desired property selected in Launch, you can create this type of rule by completing four steps:
+
+- [Create a new `Rule`](#create-a-rule)
+- [Select the `Event` you wish to trigger the rule](#select-an-event)
+- [Add any other `Conditions` required to trigger the action (optional)](#add-conditions)
+- [Select the `Action` to Attach Data and define your payload](#define-the-action)
+
+## Create a Rule
+
+In the `Rules` tab, click the `Create New Rule` button.
+
+If you don't have existing rules for this property, the button will be in the middle of the screen:
+
+<img src="../../.gitbook/assets/integrations/addFirstRule.png" width="800" /><br />
+
+If your property already has rules, the button will be in the top right of the screen:
+
+<img src="../../.gitbook/assets/integrations/addRuleFromList.png" width="800" /><br />
+
+## Select an Event
+
+You should give your rule a meaningful name so it will be easily recognizable in your list of Rules. This rule is named "Attach Places Data to Analytics Track Action Events".
```

Comment:
- Please add a comma between "name" and "so".
- Does "Rules" have to be capitalized?
- Reword the 2nd sentence, so that it reads as follows:
" In this example, the rule is named..."

- Put the name of the rule in bold --> **"Attach Places Data to Analytics Track Action Events**.